lms/models/lti_params.py

```diff
@@ -138,8 +138,28 @@ def _to_lti_v11(v13_params):
         v11_params.update({f"custom_{key}": value for key, value in custom.items()})
 
     if "roles" in v11_params:
-        # We need to squish together the roles for v1.1
-        v11_params["roles"] = ",".join(v11_params["roles"])
+        context_roles = []
+        for role in v11_params["roles"]:
+            # From: https://www.imsglobal.org/spec/lti/v1p3#role-vocabularies
```

Comment:
I read this, probably more than once, and always understood `context` as a meaningless word until just now made the context -> course association that is of course used in the rest of the document and I'm very familiar with :facepalm: